src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1887,8 +1887,7 @@ private bool AddOrganelle(OrganelleTemplate organelle)
         // 1 - you put nucleus but you already have it
         // 2 - you put organelle that need nucleus and you don't have it
```

Comment:
```suggestion
        // 2 - you put organelle that requires nucleus but you don't have one
```

Reply:
actually, this comment is a bit more outdated.
This code does not only check the uniqueness of the nucleus, it also checks the uniqueness of other organelles (currently only BA and Nucleus)

Reply:
Reworded the comment into:
```
// 1 - you put a unique organelle (means only one instance allowed) but you already have it
// 2 - you put organelle that requires nucleus but you don't have one
```